spec/template.dd

```diff
@@ -933,10 +933,46 @@ $(H4 $(LNAME2 homogeneous_sequences, Homogeneous Sequences))
         ---
         )
 
+    $(BEST_PRACTICE Prefer using a *TypeSeq* to declare variadic template
```

Comment:
Prefer a TypeSeq over what exactly?

Reply:
I also don't understand what the recommendation means.

Reply:
I've now removed this paragraph.